documents/pdf.js

```diff
@@ -0,0 +1,89 @@
+module.exports = (activities, user) => {
+  let adress2 = '';
+  if (user[0].address_of_establishment_2 !== null) {
```

Comment:
would pass a user and not an array of user, you could get rid of all the [0] in the file